rollbar_closed_channel_test.go

```diff
@@ -0,0 +1,23 @@
+package rollbar
+
+import (
+	"testing"
+)
+
+func TestDisableDefaultClient(t *testing.T) {
+	defer func() {
+		std = NewAsync("", "development", "", hostname, "")
```

Comment:
It might make sense to make a `newDefaultClient` function that returns this since this line is equivalent to one in `rollbar.go`